Dockerfile.deploy

```diff
@@ -0,0 +1,76 @@
+FROM alpine as artifacts
+
+COPY . /usr/src/
+
+RUN apk add git maven openjdk8 bash zip curl -f
+
+ENV ROOT_DIR /usr/src/
+
+WORKDIR $ROOT_DIR
+
+# Build camunda-ext
+RUN mvn clean compile package
+
+# Copy all JAR dependencies to ./target/dependencies
+RUN mvn dependency:copy-dependencies -U
+
+# Build seed
+RUN cd ./seed && \
+    ./build.sh && \
+    cd $ROOT_DIR
+
+# Build demo processes
+RUN cd ./demo_processes && \
+    find ./ -type d -maxdepth 1 -mindepth 1 -exec bash -c "cd {} && ./build.sh" ';' && \
+    cd $ROOT_DIR
+
+# Get demo processes list
+RUN cd ./demo_processes && \
+    find ./ -type d -maxdepth 1 -mindepth 1 -exec bash -c "cd {} && echo \$(basename \$(pwd))" ';' > $ROOT_DIR/demo_processes/list
+
+FROM camunda/camunda-bpm-platform:7.9.0
+
+SHELL ["/bin/bash", "-c"]
+
+USER root
+
+RUN rm -rf /camunda/webapps/camunda-invoice \
+           /camunda/webapps/examples \
+           /camunda/lib/groovy-all-2.4.13.jar \
+           /camunda/lib/mail-1.4.1.jar && \
+    apk add wget curl busybox-extras -f
+
+USER camunda
+
+RUN sed -i 's/<!-- <filter>/<filter>/' /camunda/webapps/engine-rest/WEB-INF/web.xml && sed -i 's/<\/filter-mapping> -->/<\/filter-mapping>/' /camunda/webapps/engine-rest/WEB-INF/web.xml
+
+COPY --chown=camunda:camunda ./camunda.sh /camunda/
+COPY ./context.xml /camunda/conf/
+
+COPY --from=artifacts /usr/src/target/dependencies/*.jar /camunda/lib/
+COPY --from=artifacts /usr/src/demo_processes/list /camunda/demo/war.lst
+COPY --from=artifacts /usr/src/demo_processes/*/target/*.war /camunda/webapps/
+COPY --from=artifacts /usr/src/seed/target/*.war /camunda/webapps/
+COPY --from=artifacts /usr/src/target/camunda-ext-*.jar /camunda/lib/camunda-ext.jar
+
+ENV DB_DRIVER= \
```

Comment:
What is it? Why do you need to define some environment variables without values?

Reply:
This logic is taken from previous version of Dockerfile.

In original camunda image entrypoint `camunda.sh` prefer H2 as default dataabse type, but we use Postgres. This step of building the image removes H2 specific values.

Reply:
Ok